PayrollApplicationGUI.java

```diff
@@ -1,87 +1,138 @@
 import javax.swing.*;
+import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.text.DateFormat;
 import java.text.NumberFormat;
 import java.util.Date;
 import java.util.Locale;
-import com.itextpdf.text.Paragraph;
-import com.itextpdf.text.pdf.PdfWriter;
-import java.io.FileOutputStream;
 
 public class PayrollApplicationGUI extends JFrame {
-    private JLabel idLabel, nameLabel, salaryLabel, companyNameLabel, companyAddresslabel, dateLabel;
+    private JLabel idLabel, nameLabel, salaryLabel, companyInfoLabelName,companyInfoLabelAddress,companyInfoLabelEmail ,dateLabel;
     private JTextField idField, nameField, salaryField;
-    private JButton calculateButton, generatePDFButton;
-    private JTextArea resultArea;
-    private Locale locale;
+    private JButton calculateButton;
+    private JLabel grossSalaryLabel, hraLabel, taLabel, maLabel, daLabel, pfLabel, taxLabel, netSalaryLabel;
 
     public PayrollApplicationGUI() {
         setTitle("Payroll Application");
-        setSize(600, 500);
+        setSize(800, 700);
```

Comment:
size of the frame looks too large, set it according to the content in the frame.